_quarto.yml

```diff
@@ -6,15 +6,23 @@ website:
   bread-crumbs: false
   favicon: static/bsa-favicon.png
   title: "Producing Quality Analysis" 
+  # Search is good to have and works very well. But it introduces an accessibility
+  # issue no matter how it is configured.
+  # https://quarto.org/docs/websites/website-search.html
+  # overlay gives empty button, while textbox gives empty label
+  # Disabled for now until a solution is found
+  search: false
```

Comment:
Although it says "until a solution is found", we have not yet looked at this again. I do remember spending a couple of hours trying to get this accessible, so not sure it is worth further time spent for the added functionality.